path/src/keybindings.ps1

```
@@ -0,0 +1,251 @@
+function script:Read-FountEditorKeybindings([string]$Path) {
+	if (-not (Test-Path $Path)) { return @() }
+	try {
+		$parsed = Get-Content $Path -Raw -Encoding UTF8 | ConvertFrom-Json -Depth 20
+	}
+	catch { return @() }
+	if ($parsed -is [System.Array]) { return $parsed }
+	if ($parsed.PSObject.Properties['keybindings']) { return @($parsed.keybindings) }
+	return @()
+}
+
+function script:Merge-FountEditorKeybindings([string]$KeybindingsPath) {
```

PSScriptAnalyzer (1.25.0)

[warning] The cmdlet 'script:Merge-FountEditorKeybindings' uses a plural noun. A singular noun should be used instead.

Suggested fix: Singularized correction of 'script:Merge-FountEditorKeybindings'

(PSUseSingularNouns)